Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -52,6 +52,62 @@ class RemoteAnimationStack;
 class RemoteAnimationTimeline;
 #endif
 
+enum class PendingCommitMessage {
```

Comment:
You may want to specify the storage type to be uin8_t for these new enums, it should reduce their size from 4 to 1 byte, and struct PendingCommit may shrink a bit as well (but not sure, depending on TransactionID's alignment).
Unless you see perf reasons to keep them wide and int-aligned?